package.json

```diff
@@ -30,16 +30,21 @@
     "ember-cli": "^2.4.2",
     "ember-cli-blueprint-test-helpers": "^0.12.0",
     "ember-cli-dependency-checker": "^1.2.0",
+    "ember-cli-internal-test-helpers": "^0.7.0",
+    "ember-cli-is-package-missing": "^1.0.0",
+    "ember-router-generator": "^1.2.1",
```

Comment:
> I'm wondering how blueprint tests can run without these. Is there a place where test only dependencies are specified? Or could it be that I'm running npm v2 and thus not picking up ember-cli dependencies in root of node_modules?

npm@2 could indeed be the reason why it's failing for you. good catch anyway, would be cool though if you could extract that dependency fix into another PR or at least commit.